Makefile

```diff
@@ -236,6 +236,9 @@ remove_unused_imports: $(PYSOURCES)
 	autoflake --in-place --remove-all-unused-imports $^
 	make format
 
+remove_trailing_whitespace:
+	$(CURDIR)/contrib/admin/remove_trailing_whitespace.py
+
```

Comment:
This is good!

I also meant adding the path to the `mypy` checker on https://github.com/DataBiosphere/toil/blob/fde82bca13faa979b921e76f9801737258420ef6/Makefile#L248

Reply:
@mr-c Since I wanted to create a mypy check for all of the currently passing files, and include it as a part of our CI/CD, I've opened a separate PR for this: https://github.com/DataBiosphere/toil/pull/3458

If that's approved and merged, I'll do a pull from master here and it should be included.  I'll probably also add a flake8 check explicitly for trailing whitespace.